bitnami/grafana-alloy/values.yaml

```diff
@@ -0,0 +1,931 @@
+## @skip alloy.image.tag Grafana Alloy image tag (immutable tags are recommended)
+## @param alloy.image.pullPolicy Grafana Alloy image pull policy
+## @param alloy.image.pullSecrets Grafana Alloy image pull secrets
+## @param alloy.image.debug Enable image debug mode
+##
+  image:
+    registry: docker.io
+    repository: bitnami/grafana-alloy
+    tag: 1.8.3-debian-12-r0
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+    debug: false
+  ## @param alloy.containerPorts.http Grafana Alloy HTTP container port
+  ##
+  containerPorts:
+    http: 8080
+  ## @param alloy.extraContainerPorts Optionally specify extra list of additional ports for Grafana Alloy containers
+  ## e.g:
+  ## extraContainerPorts:
+  ##   - name: myservice
+  ##     containerPort: 9090
+  ##
+  extraContainerPorts: []
+  ## @param alloy.existingConfigmap The name of an existing ConfigMap with your custom configuration for Grafana Alloy
+  ##
+  existingConfigmap: ""
+  ## @param alloy.configuration Specify content for Grafan Alloy config file. Omitted if alloy.existingConfigmap is provided.
+  ## e.g:
+  ## configuration: |-
+  ##   ...
+  ##
+  configuration: ""
+  ## @param alloy.extraConfig Append extra configuration to the default config file
+  ##
+  extraConfig: ""
+  ## @param alloy.resourceType Type of controller to use for deploying Grafana Alloy in the cluster.
+  ## Must be one of 'daemonset', 'deployment', or 'statefulset'.
+  ##
+  resourceType: 'daemonset'
+
+  ## @param alloy.replicaCount Number of pods to deploy. Ignored when alloy.resourceType is 'daemonset'.
+  ##
+  replicaCount: 1
+  ## @param alloy.clustering.enabled Deploy Grafana Alloy in a cluster to allow for load distribution.
+  ## @param alloy.clustering.name Name for the Grafana Alloy cluster. Used for differentiating between clusters.
+  ## @param alloy.clustering.portName Name for the port used for clustering, useful if running inside an Istio Mesh
+  ##
+  clustering:
+    enabled: false
+    name: ""
+    portName: http
+
+  ## @param alloy.stabilityLevel Minimum stability level of components and behavior to enable. Must be
+  ##  one of "experimental", "public-preview", or "generally-available".
+  stabilityLevel: "generally-available"
+
+  ## @param alloy.listenAddr Address to listen for traffic on. 0.0.0.0 exposes the UI to other containers.
+  listenAddr: 0.0.0.0
+
+  ## @param alloy.uiPathPrefix Base path where the UI is exposed.
+  uiPathPrefix: /
```

Comment:
Does this affect metrics endpoints? I mean, if you customize this, should the `/metrics` endpoint used on ServiceMonitor and other objects be adapted?